packages/terra-form-select/src/shared/_FrameUtil.js

```diff
@@ -96,13 +100,13 @@ class FrameUtil {
    * @param {ReactNode} dropdown - The component dropdown.
    * @return {boolean} - True if the dropdown should be positioned.
    */
-  static shouldPositionDropdown(previousState, currentState, dropdown) {
+  static shouldPositionDropdown(previousState, currentState, dropdown, isTouchAccessible) {
     if (!currentState.isOpen) {
       return false;
     }
 
     const { bottom } = dropdown.getBoundingClientRect();
-    return previousState.isOpen === false || bottom > window.innerHeight;
+    return previousState.isOpen === false || bottom > window.innerHeight || (!previousState.resizeOnSearch && currentState.resizeOnSearch && isTouchAccessible);
```

Comment:
Why do we need both the conditions here `!previousState.resizeOnSearch && currentState.resizeOnSearch` why not only this `(currentState.resizeOnSearch && isTouchAccessible)` ?

Reply:
Removed 07f81a8